pages/docs/reference/whatsnew14.md

```diff
@@ -6,6 +6,19 @@ title: "What's New in Kotlin 1.4"
 
 # What's New in Kotlin 1.4
 
+## Delegated properties improvements
+
+In 1.4, we add new features to improve your experience with delegated properties in Kotlin:
```

Comment:
In 1.4, we have added new features to improve your experience with delegated properties in Kotlin:

is it "delegated properties" or "delegate properties"?

Reply:
it's "delegated properties"